### PR TITLE
Fixed #25371 -- Added reverse_sql and reverse_code examples to docs.

### PR DESCRIPTION
**Pull request by iambibhas**
Ticket - https://code.djangoproject.com/ticket/25371

### CODE REVIEW

File: docs/ref/migration-operations.txt [hunk 1]
```diff
@@ -224,6 +224,14 @@ queries and parameters in the same way as :ref:`cursor.execute()
 If you want to include literal percent signs in the query, you have to double
 them if you are passing parameters.
 
+The ``reverse_sql`` statement is executed when the migration is reversed. So you can
```
Comment (MarkusH) [inline docs/ref/migration-operations.txt]: Can you limit the line length for text to 79 chars, please.

Comment (MarkusH) [inline docs/ref/migration-operations.txt]: "... the migration is _unapplied_. So you .."

File: docs/ref/migration-operations.txt [hunk 2]
```diff
@@ -265,6 +273,10 @@ match the operation's place in the project history, and the second is an
 instance of :class:`SchemaEditor
 <django.db.backends.base.schema.BaseDatabaseSchemaEditor>`.
 
+The ``reverse_code`` argument is called when running migrations backward.
```
Comment (MarkusH) [inline docs/ref/migration-operations.txt]: "... when unapplying migrations."

**Comment by iambibhas** [comment]
@MarkusH made the changes.

**Comment by MarkusH** [comment]
LGTM

File: docs/ref/migration-operations.txt [hunk 3]
```diff
@@ -224,6 +224,14 @@ queries and parameters in the same way as :ref:`cursor.execute()
 If you want to include literal percent signs in the query, you have to double
 them if you are passing parameters.
 
+The ``reverse_sql`` statement is executed when the migration is unapplied. So
+you can reverse the changes done in the first statement::
```
Comment (timgraham) [inline docs/ref/migration-operations.txt]: first statement -> forwards queries?

File: docs/ref/migration-operations.txt [hunk 4]
```diff
@@ -224,6 +224,14 @@ queries and parameters in the same way as :ref:`cursor.execute()
 If you want to include literal percent signs in the query, you have to double
 them if you are passing parameters.
 
+The ``reverse_sql`` statement is executed when the migration is unapplied. So
```
Comment (timgraham) [inline docs/ref/migration-operations.txt]: "statement is" -> "queries are"?
, so you can... (one sentence)?

File: docs/ref/migration-operations.txt [hunk 5]
```diff
@@ -294,6 +306,14 @@ model::
             Country(name="France", code="fr"),
         ])
 
+    def backwards_func(apps, schema_editor):
+        # forwards_func creates two Country instance,
```
Comment (timgraham) [inline docs/ref/migration-operations.txt]: instances

File: docs/ref/migration-operations.txt [hunk 6]
```diff
@@ -294,6 +306,14 @@ model::
             Country(name="France", code="fr"),
         ])
 
+    def backwards_func(apps, schema_editor):
```
Comment (timgraham) [inline docs/ref/migration-operations.txt]: maybe call it `reverse_func` to parallel the `reverse_code` parameter name?

File: docs/ref/migration-operations.txt [hunk 7]
```diff
@@ -301,6 +321,7 @@ model::
         operations = [
             migrations.RunPython(
```
Comment (timgraham) [inline docs/ref/migration-operations.txt]: I'd consolidate this on one line: `migrations.RunPython(forwards_func, reverse_func)`

File: docs/ref/migration-operations.txt [hunk 8]
```diff
@@ -265,6 +273,10 @@ match the operation's place in the project history, and the second is an
 instance of :class:`SchemaEditor
 <django.db.backends.base.schema.BaseDatabaseSchemaEditor>`.
 
+The ``reverse_code`` argument is called when unapplying migrations. This
+callable is used so that the migration is reversible by undoing what is being
```
Comment (timgraham) [inline docs/ref/migration-operations.txt]: This callable should undo what is done in the ``code`` callable so that the migration is reversible.

**Comment by timgraham** [comment]
You can squash commits when updating, thanks.

**Comment by iambibhas** [comment]
@timgraham looks like I screwed up while rebasing. Should I close this PR and create another?

**Comment by timgraham** [comment]
I think you can fix it without creating a new pull request.

**Comment by iambibhas** [comment]
@timgraham I think it's fixed.

**Comment by timgraham** [comment]
merged in 4283a038431ef6428d086d4911179bd8eb8b2299, thanks!

**Pull request closed without merging**